src/utils/apis.js

```diff
@@ -22,6 +21,17 @@ export function getArticles(params) {
     .catch((err) => console.log(err, 'all articles'));
 }
 
+export function getSingleArticle({article_id}) {
+  return axios
+  .get(`${baseURL}articles/${article_id}`)
+  .then(({ data }) => {
```

Comment:
As we aren't processing the data in anyway here in the .then and just returning it, we don't need the .then(). just return the axios.get